### PR TITLE
*: Tiny clean up

### PR DESCRIPTION
**Pull request by zimulala**

### CODE REVIEW

File: kv/iter.go [hunk 1]
```diff
@@ -65,7 +65,7 @@ func DecodeValue(data []byte) ([]interface{}, error) {
 }
 
 // NextUntil applies FnKeyCmp to each entry of the iterator until meets some condition.
-// It will stop when fn returns true, or iterator is invalid or occur error
+// It will stop when fn returns true, or iterator is invalid or occur error.
```
Comment (shenli) [inline kv/iter.go]: occur error -> any error occurs

Comment (zimulala) [inline kv/iter.go]: occur error -> occurs errors?

Comment (shenli) [inline kv/iter.go]: "error" is the subject and "occur" is the predicate. 
occur is an intransitive verb.

Comment (zimulala) [inline kv/iter.go]: got it.

File: kv/index_iter.go [hunk 2]
```diff
@@ -152,7 +152,7 @@ func (c *kvIndex) GenIndexKey(indexedValues []interface{}, h int64) (key []byte,
 }
 
 // Create creates a new entry in the kvIndex data.
-// If the index is unique and there already exists an entry with the same key, Create will return ErrKeyExists
+// If the index is unique and it already existed an entry with the same key, Create will return ErrKeyExists.
```
Comment (qiuyesuifeng) [inline kv/index_iter.go]: How about `there is an existed entry`?

File: kv/iter.go [hunk 3]
```diff
@@ -65,7 +65,7 @@ func DecodeValue(data []byte) ([]interface{}, error) {
 }
 
 // NextUntil applies FnKeyCmp to each entry of the iterator until meets some condition.
-// It will stop when fn returns true, or iterator is invalid or occur error
+// It will stop when fn returns true, or iterator is invalid or occur errors.
```
Comment (siddontang) [inline kv/iter.go]: occur errors -> occurs an error. 
But I prefer occurs -> raises

Comment (qiuyesuifeng) [inline kv/iter.go]: I think `an error occurred` is better.

Comment (shenli) [inline kv/iter.go]: @siddontang occur is an intransitive verb.

File: kv/index_iter.go (hunk 2 above)
Comment (zimulala) [inline kv/index_iter.go]: it‘s better.

File: kv/iter.go (hunk 3 above)
Comment (zimulala) [inline kv/iter.go]: How about `an error occurs`?

Comment (zimulala) [inline kv/iter.go]: How about `an error occurs`? @siddontang @qiuyesuifeng @shenli

Comment (siddontang) [inline kv/iter.go]: yep

**Comment by shenli** [comment]
LGTM

**Comment by zimulala** [comment]
PTAL @siddontang

**Pull request merged**